keyvalue/clients/src/main/java/tachyon/client/keyvalue/KeyValueStoreWriter.java

```diff
@@ -48,4 +51,15 @@
    */
   void put(ByteBuffer key, ByteBuffer value) throws IOException, TachyonException;
 
+  /**
```

Comment:
I'm going to delete this API, and replace its usage in OutputCommitter with a better strategy.